### DISCUSSION TITLE
Вопрос выбивания Ягод Гоблина / Драконьих корней

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#69] [General]
Проверить выбиваются ли корни дракона и ягоды гоблина из монстров.

в Готике 2 оригинал выбить можно было, а в Ночь Ворона возможность выбивать из монстров пофиксили, переработав момент генерации ништяков

**Comment by Kor-Angar** [comment]
проверил Палов. "выбить" из них ягодки нельзя. генерятся при создание мира(так правильно писать?).

**Comment by Kor-Angar** [comment]
сегодня еще проверю чёрных гоблинов и элитных орков.

**Comment by dosinabox** [comment]
> проверил Палов. "выбить" из них ягодки нельзя.

Это правильно или неправильно? По сравнению с чистыми декомпилированными скриптами 1.3 это никак не менялось.

> генерятся при создание мира(так правильно писать?).

Да, но правильнее - "при вставке персонажа в мир".

**Comment by Kor-Angar** [comment]
я протестирую гоблинов и орков и отпишусь. пока непонятно

**Comment by dosinabox** [comment]
> я протестирую гоблинов и орков и отпишусь. пока непонятно

Ну как там?)

**Comment by Kor-Angar** [comment]
из Гоблинов и Элитных орков можно "выбивать" ягоды гоблина и драконьи корни как в Готике 2 версии 1,30

не знаю какой из этого следует вывод, но в вопросе разобрались хотя бы в том плане что из Паладинов нельзя было выбить ягодки ни в Готике 2 классик ни в Готике 2 НВ. Поэтому возможно Пираньи вовсе не забыли про Ягодки из Паладинов в НВ, как часто пишет Димус, а оставили эту историю сознательно.

надо будет ещё раз проверить на 1,30 выбиваются ли из палов ягодки:-))

**Comment by dosinabox** [comment]
> надо будет ещё раз проверить на 1,30 выбиваются ли из палов ягодки:-))

Как дела?)

**Comment by Kor-Angar** [comment]
что то из Палов ягодки не падают. ты ничего не правила?

**Comment by Kor-Angar** [comment]
> что то из Палов ягодки не падают. ты ничего не правила?

в минентале

**Comment by Kor-Angar** [comment]
у Удара одна была. может по вероятности так сказать. буду еще проверять :)

**Comment by dosinabox** [comment]
Это не менялось. Просто не повезло)

**Comment by Kor-Angar** [comment]
у черных гоблинов рандомно появляются в Хориносе +
